R/Corin.R

```diff
@@ -0,0 +1,32 @@
+#' @md
+#' @name Corin
+#' @title Daily dataset for the Corin catchment.
+#' @description A daily dataset containing precipitation, streamflow, potential
+#' evapotranspiration and average temperature for the Corin catchment (147 km²),
+#' ACT, Australia. The Corin catchment is the headwaters of the greater Cotter
+#' River catchment, which is a tributary to the Murrumbidgee River.
+#' @usage data("Corin")
+#' @format
+#' A `zoo` object of class `"zooreg", "zoo"`. It is a regular time series
+#' indexed by days, in `Date` format.
+#' There are four columns:
+#' * `P` (precipitation, mm/day)
+#' * `Q` (streamflow, mm/day)
+#' * `E` (potential evapotranspiration, mm/day)
+#' * `T` (daily average temperature, ºC)
+#' @details
+#' * Daily rainfall at Cotter Hut (ID: 570946).
+#' * Daily mean streamflow at Cotter River at Gingera (ID: 410730).
```

Comment:
This this seems to be the same gauge as ?Cotter
I'm happy merging this for the time being, but before the 1.0 release we need to rationalise the datasets included #72

Reply:
Indeed it is the same gauge. I didn't want to modify the existing dataset in case it broke something for the time being, but I can revisit this. 

Maybe adding temperature data to the existing Cotter dataset would be easier.

Reply:
Yep, fine like this.
I've added a comment to #72 . It'll be good to get your input into that when the time comes